CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 <!-- 10.1.2 START -->
 
+# 10.1.2 (2023-09-21)
```

Comment:
You can remove this, changelog is auto-generated using the `add to changelog` label.